lib/rspec/matchers/built_in/be.rb

```diff
@@ -151,13 +152,15 @@ def matches?(actual)
         # @api private
         # @return [String]
         def failure_message
-          "expected: #{@operator} #{expected_formatted}\n     got: #{@operator.to_s.gsub(/./, ' ')} #{actual_formatted}"
+          "expected: #{@operator} #{expected_formatted}\n     " \
+          "got: #{@operator.to_s.gsub(/./, ' ')} #{actual_formatted}"
```

Comment:
This string is designed so that the expected and actual ("got") value align, so it'd be nice to format the source string that way, too:

``` ruby
"expected: #{@operator} #{expected_formatted}\n" \
"     got: #{@operator.to_s.gsub(/./, ' ')} #{actual_formatted}"
```